conda_forge_tick/migrators.xsh

```diff
@@ -337,6 +337,24 @@ class Version(Migrator):
                 n = eval_version(n)
                 replace_in_file(p, n, f)
             self.set_build_number('meta.yaml')
+
+        # Set the provider to Azure only
```

Comment:
You are adding this to the Version updater. You need to create a new migrator subclassing Rebuild migrator and override its `migrate` method